src/components/dialog/dialog.component.js

```diff
@@ -34,12 +36,20 @@ const Dialog = ({
   disableClose,
   help,
   role = "dialog",
+  contentPadding = {},
   ...rest
 }) => {
   const dialogRef = useRef();
   const innerContentRef = useRef();
   const titleRef = useRef();
   const listenersAdded = useRef(false);
+  const { p, py, px } = contentPadding;
+
+  const paddingTopInnerContent = [py, p].some(
+    (padding) => padding !== undefined
+  )
+    ? `${ZERO_PADDING}px`
```

Comment:
I think that should be just 0 (without px), and we don't need to set it to a const as it's pretty self explanatory